docs/s3.md

```diff
@@ -0,0 +1,72 @@
+# Performance Test Result storage in S3 bucket for SLV
+
+**NOTE: This document is only relevant to puppet employees**
+
+The SLV team has an s3 bucket for storing results from performance testing.
+
+`
+s3://slv-performance-results
+`
+
+## Convention
+
+S3 is [object storage](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html),
+so it doesn't have true directories.  But it does like to make it look like it does.  The convention 
+is to use names like `foo/bar`  and `foo/car` to group files together.  They are really two separate
+file that exist next to each other, but with the cli you can `ls foo/` and you will get two
```

Comment:
Should have been more specific; the "file" in "They are really two separate file that exist next to each other" should be "files" :)

Reply:
lol, maybe I am too tired to be doing this... :)   More caffeine needed. :)